prime-router/src/main/kotlin/cli/LookupTableCommands.kt

```diff
@@ -639,9 +653,10 @@ class LookupTableCompareMappingCommand : GenericLookupTableCommand(
 /**
  * Create a new lookup table.
  */
-class LookupTableCreateCommand : GenericLookupTableCommand(
+class LookupTableCreateCommand(httpClient: HttpClient? = null) : GenericLookupTableCommand(
```

Comment:
Can we specify the `httpClient` here?  These are actual CLI command classes so I'm not sure we would necessarily want a new version with a custom client?